test/end-to-end/test/test_pendingCommit.js

```diff
@@ -95,7 +95,7 @@ describe('Changes Pending Commit Page', () => {
     const testSpec3 = test('should show correct pending change content',
       (done) => {
         // Highlight the expected result
-        const expectedAction = 'Removed';
+        const expectedAction = 'Added';
```

Comment:
Is that because on the previous bdcs-rs backend we had a bunch of predefined recipes, but the haskell one doesn't?

Reply:
Nope, the Haskell backend also uses the same predefined recipes. They are bind-mounted to the backend container. 

From what I can tell this test has been just broken. Maybe the call to `updateComponent` has something to do with revealing this issue. I don't really know.